src/main/java/com/box/sdk/BoxDeveloperEditionAPIConnection.java

```diff
@@ -201,19 +209,20 @@ public void refresh() {
         this.getRefreshLock().writeLock().unlock();
     }
 
-    String jwtConstructAssertion() {
+    private String constructJWTAssertion() {
         JwtClaims claims = new JwtClaims();
         claims.setIssuer(this.getClientID());
-        claims.setAudience(this.getTokenURL());
-        claims.setExpirationTimeMinutesInTheFuture(0.2f);
+        claims.setAudience("https://api.box.com/oauth2/token");
```

Comment:
I wanted to keep it different from the TOKEN_URL_STRING member of the BoxAPIConnection class. That will become an issue if the token url is changed (e.g. to point to test environment) but the audience still needs to be the same. Hence ended up just hardcoding. But I understand the point. Will move it to a private static member in BoxDeveloperEditionAPIConnection and will use that instead.